src/components/RangeFacet.tsx

```diff
@@ -12,7 +12,7 @@ class RangeFacet extends React.Component<PropsType, State> {
     render() {
         const facet = this.props.facet as Store.RangeFacet;
         let css = objAssign({}, defaultCss, this.props.css);
-        const { onRangeChange, afterRangeChange, loadedResultsCount } = this.props;
+        const { onRangeChange, afterRangeChange, loadedResultsCount, resultCount } = this.props;
```

Comment:
Just foreseeing future trouble here, and it doesn't really have much bearing on this review but resultsCount may never change if user doesn't specify { count: true } in the searchParameters. Since we're introducing this property, maybe we should add a comment that gives this context so that it's clear for future changes.